.changeset/few-stingrays-lay.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nextui-org/user": minor
+---
+
+Updated avatarProps of use-user function, to only pass the name(of avatarProps), when no avatar icon source passed
```

Comment:
please change it to

```
Removed `name` from `avatarProps` in `use-user.ts` (#3369)
```

Reply:
Kindly explain, how can  do that ?
Commands